src/test/java/uk/gov/homeoffice/digital/sas/balancecalculator/BalanceCalculatorIntegrationTest.java

```diff
@@ -64,4 +65,33 @@ void calculate_endToEnd_contributionsAndCumulativeTotalsAsExpected() {
     assertThat(accruals.get(3).getCumulativeTotal())
         .usingComparator(BigDecimal::compareTo).isEqualTo(BigDecimal.valueOf(8160));
   }
+
+  @Test
+  void calculate_timeEntryHasTwoDaysSpan_contributionsAndCumulativeTotalsAsExpected() {
```

Comment:
Is it worth adding test cases to this to cover more scenarios e.g. boundaries on days, GMT vs BST and periods that cross the two.

Reply:
We have added an extra integration test so that we are testing the scenario where GMT changes to BST and vice versa

Reply:
We are currently testing boundaries across days in the `calculate_timeEntryHasTwoDaysSpan_contributionsAndCumulativeTotalsAsExpected()` test

Reply:
Great